regimes/mx/complements/fuel_complement.go

```diff
@@ -0,0 +1,58 @@
+// Package complements provides GOBL Complements for Mexican invoices
+package complements
+
+import (
+	"cloud.google.com/go/civil"
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+)
+
+// FuelComplement provides detailed information about fuel purchases made with
+// electronic wallets. In Mexico, e-wallet suppliers are required to report this
+// information as part of the invoices they issue to their customers.
+type FuelComplement struct {
+	// Customer's e-wallet account number
+	WalletAccountNumber string `json:"wallet-account-number" jsonschema:"title=Wallet Account Number"`
+	// List of fuel purchases made with the customer's e-wallets
+	Lines []*FuelLine `json:"lines" jsonschema:"title=Lines"`
+	// Summary of all the purchases totals, including taxes (calculated)
+	Totals bill.Totals `json:"totals" jsonschema:"title=Totals" jsonschema_extras:"calculated=true"`
+}
+
+// FuelLine represents a single fuel purchase made with an e-wallet issued by
+// the invoice's supplier to the invoice's customer.
+type FuelLine struct {
+	// Identifier of the e-wallet used to make the purchase
+	WalletID cbc.Code `json:"wallet-id" jsonschema:"title=Wallet Identifier"`
+	// Date and time of the purchase
+	PurchaseDateTime civil.DateTime `json:"purchase-date" jsonschema:"title=Purchase Date"`
```

Comment:
We don't have `cal.DateTime` at the moment. Should we add it?

Reply:
🤔 interesting. It might indeed be a good idea to have a `cal.DateTime` which enforces ISO formatting. We have a similar "at" package in github.com/invopop/lib which does something similar with database storage. I'll think a bit more about this.

Reply:
Ok! Leaving `civil.DateTime` for now

Reply:
I just pushed 5a3388e with a barebones `cat.DateTime` implementation (I'll completely once we confirm that's the like we'd like to follow). The only concern I have about the type is whether or not it should support time zones. The CFDI expects a date and time **without** a timezone, but in the future we may need a Date and Time with Timezones for other purposes. Not sure we'll want to add just another type then.